docs/migrations.md

```diff
@@ -401,536 +403,59 @@ export class QuestionRefactoringTIMESTAMP implements MigrationInterface {
 }
 ```
 
----
```

Comment:
Hi! I see you added new functionality for migrations, but you haven't deleted the old functionality, so as far as I understand, old documentation is still relevant. Could you please explain your motivation behind removing this entire chunk of documentation?